Test/Unit/Controller/Cart/EmailTest.php

```diff
@@ -130,11 +139,12 @@ public function testExecute_noEmail()
     }
 
     /**
-     * This doesn't really seem to test invalid emails as the validateEmail method
-     * needs to be mocked. Generally just making sure that if validateEmail comes
-     * back false we error handle properly
+     * @test
      */
-    public function testExecute_invalidEmail()
+    // Note: This doesn't really seem to test invalid emails as the validateEmail method
```

Comment:
Can you bubble this comment into the phpdoc?

Reply:
I actually moved it form php doc to comment - I don't think this should live in PHPDoc. PHP doc is for public API documentation. This is more about note around how this function works internally

Reply:
Fair enough 👍